src/components/header.tsx

```diff
@@ -35,7 +35,7 @@ const NewHeader = (): JSX.Element => {
             <div className='hidden md:block w-full max-w-max justify-items-end'>
               <div className='ml-10 flex items-baseline space-x-4'>
                 <a className='cursor-pointer hover:text-purple-500 capitalize font-medium'>
-                  <Link href='/comandos'>{t('commands')}</Link>
+                  <Link href='/commands'>{t('commands')}</Link>
```

Comment:
```suggestion
<Link href='/comandos'>{t('commands')}</Link>
```
The route is `comandos`

Reply:
I think we should change all routes to english, but this in another pr

Reply:
ok